src/main/java/org/tron/common/storage/DepositImpl.java

```diff
@@ -302,7 +302,14 @@ public synchronized Storage getStorage(byte[] address) {
 
     Storage storage;
     if (this.parent != null) {
-      storage = parent.getStorage(address);
+      Storage parentStorage = parent.getStorage(address);
+      if (Args.getInstance().getBlockVersion() == 3) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Assign this magic number 3 to a well-named constant, and use the constant instead. ([squid:S109](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS109))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6pnSgN4uCclo1ct&open=AWbJU6pnSgN4uCclo1ct)

[ISSUE_KEY]: AWbJU6pnSgN4uCclo1ct